src/libraries/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs

```diff
@@ -1993,5 +1994,24 @@ public void CheckSignatureMultipleEnvelopedSignatures(string signatureParent/*,
 
             Assert.False(subject.CheckSignature());
         }
+
+        private static Dictionary<string, string> EnvelopedSignatureWithRootXpointerReference = new ()
```

Comment:
This dictionary is could be `readonly` and doesn't follow the static naming convention. It should be `s_envelopedSignatureWithRootXpointerReference`.

That said, maybe this would look better as a theory using `MemberData` instead of a dictionary as a lookup. What do you think?